README.md

```diff
@@ -128,74 +151,63 @@ for more details.
+a Python version greater than 2.7 or 3.4 (Python <=3.3 is no longer supported).
 
 ### Runtime dependencies
 
-* **Python** 2.7 or 3.3+
-* **PyQt5** 5.2+ or **PyQt4** 4.6+: PyQt5 is recommended.
+* **Python** 2.7 or 3.4+: The core language Spyder is written in and for.
+* **PyQt5** 5.2+: Python bindings for Qt, used for Spyder's GUI.
 * **qtconsole** 4.2.0+: Enhanced Python interpreter.
 * **Rope** 0.9.4+ and **Jedi** 0.9.0+: Editor code completion, calltips
   and go-to-definition.
 * **Pyflakes**: Real-time code analysis.
 * **Sphinx**: Rich text mode for the Help pane.
 * **Pygments** 2.0+: Syntax highlighting for all file types it supports.
 * **Pylint**: Static code analysis.
-* **Pycodestyle**: Style analysis.
+* **Pycodestyle**: Real-time code style analysis.
 * **Psutil**: CPU and memory usage on the status bar.
 * **Nbconvert**: Manipulation of notebooks in the Editor.
 * **Qtawesome** 0.4.1+: To have an icon theme based on FontAwesome.
 * **Pickleshare**: Show import completions on the Python consoles.
 * **PyZMQ**: Run introspection services asynchronously.
-* **QtPy** 1.2.0+: Abstracion layer for Python Qt bindings so that Spyder can run on PyQt4
-  and PyQt5.
+* **QtPy** 1.2.0+: Abstraction layer for Python Qt bindings so that Spyder
+  can run on multiple Qt bindings and versions.
 * **Chardet**: Character encoding auto-detection in Python.
-* **Numpydoc**: Used by Jedi to get return types for functions with Numpydoc docstrings.
-* **Cloudpickle**: Serialize variables in the IPython kernel to send them to Spyder.
+* **Numpydoc**: Used by Jedi to get function return types from Numpydocstrings.
+* **Cloudpickle**: Serialize variables in the IPython kernel to send to Spyder.
 
 ### Optional dependencies
 
 * **Matplotlib**: 2D/3D plotting in the Python and IPython consoles.
 * **Pandas**: View and edit DataFrames and Series in the Variable Explorer.
-* **Numpy**: View and edit two or three dimensional arrays in the Variable Explorer.
+* **Numpy**: View and edit 2- or 3-dimensional arrays in the Variable Explorer.
 * **SymPy**: Symbolic mathematics in the IPython console.
 * **SciPy**: Import Matlab workspace files in the Variable Explorer.
 * **Cython**: Run Cython files in the IPython console.
 
 
-## Contributing
-
-Everyone is welcome to help with Spyder. Please read our
-[contributing instructions](https://github.com/spyder-ide/spyder/blob/master/CONTRIBUTING.md)
-to get started!
-
-
-## Troubleshooting
-
-Before posting a report, *please* carefully read our **[Troubleshooting Guide](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ)**
-and search the [issue tracker](https://github.com/spyder-ide/spyder/issues)
-for your error message and problem description, as the great majority of bugs
-are either duplicates, or can be fixed on the user side with a few easy steps.
-Thanks!
+## More information
 
+[Main Website](https://spyder-ide.github.io/)
```

Comment:
Please remove this link for now. @goanpeca bought spyder-ide.org some time ago, and that's the URL we should promote for our website. not this one.

Reply:
Oh okay; will do.